steamworks.go

```diff
@@ -114,4 +118,7 @@ const (
 	flatAPI_SteamUtils                               = "SteamAPI_SteamUtils_v010"
 	flatAPI_ISteamUtils_IsSteamRunningOnSteamDeck    = "SteamAPI_ISteamUtils_IsSteamRunningOnSteamDeck"
 	flatAPI_ISteamUtils_ShowFloatingGamepadTextInput = "SteamAPI_ISteamUtils_ShowFloatingGamepadTextInput"
+
+	flagAPI_SteamFriends                 = "SteamAPI_SteamFriends_v017"
```

Comment:
Please keep the alphabetical order. So, this should be between the flatAPI_SteamApps group and the flatAPI_SteamInput group.